runbotics/runbotics-orchestrator-ui/src/src-landing/components/LanguageSwitcher/LanguageSwitcher.module.scss

```diff
@@ -0,0 +1,53 @@
+@import '#src-landing/scss/main';
+@import '#src-landing/components/Typography/Typography.module.scss';
+
+.option {
+    font-family: 'Montserrat';
+    font-size: large;
+    font-weight: 500;
+    padding: 10px;
+    cursor: pointer;
+}
+.selectWrapper {
+    position: relative;
+    width: 40px;
+    aspect-ratio: 1/1;
+    display: flex;
+    border: 2px solid black;
```

Comment:
dont make border permanent, imo its better to manipulate the background color like search button on blog page. default background is transparent but on hover make it darker than the background. Every color you need is in palette scss